test/libs/metrics/top_level_metrics_test.py

```diff
@@ -101,24 +173,25 @@ def test_top_level_metrics_basic():
         "2020-08-19,36,,,,,3,10,20,\n"
         "2020-08-20,36,40,,40,360,4,10,20,\n"
     )
-    one_region = _fips_csv_to_one_region(data, Region.from_fips("36"))
     latest = {
         CommonFields.POPULATION: 100_000,
         CommonFields.FIPS: "36",
         CommonFields.STATE: "NY",
         CommonFields.ICU_TYPICAL_OCCUPANCY_RATE: 0.5,
         CommonFields.ICU_BEDS: 30,
     }
-    one_region = dataclasses.replace(one_region, latest=latest)
+    one_region = _fips_csv_to_one_region(data, Region.from_fips("36"), latest=latest)
     results, _ = top_level_metrics.calculate_metrics_for_timeseries(
         one_region, None, None, structlog.get_logger(), require_recent_icu_data=False
     )
 
     expected = _build_metrics_df(
-        "2020-08-17,36,10,,0.02,,,0.5\n"
-        "2020-08-18,36,10,0.1,0.04,,,0.5\n"
-        "2020-08-19,36,,0.1,,,,0.5\n"
-        "2020-08-20,36,,0.1,,,,0.5\n"
+        "36",
+        start_date="2020-08-17",
```

Comment:
I'm not sure this treatment makes sense for all existing literal CSVs in test code but it sure makes these easier to read.